### PR TITLE
Update python-base submodule and re-generated client to 10.0.0-snapshot

### PR DESCRIPTION
**Pull request by roycaihw**
in preparation of 10.0.0a1 release

/assign @yliaog

### CODE REVIEW

**Comment by roycaihw** [comment]
the majority of line changes comes from
```
kubernetes/swagger.json.unprocessed                | 150161 +++++++++---------
```

which was entirely sorted in https://github.com/kubernetes/kubernetes/pull/71223

File: README.md [hunk 1]
```diff
@@ -83,18 +83,19 @@ supported versions of Kubernetes clusters.
 
 #### Compatibility matrix
 
-|                    | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 | Kubernetes 1.8 | Kubernetes 1.9 | Kubernetes 1.10 | Kubernetes 1.11 | Kubernetes 1.12 | Kubernetes 1.13 |
-|--------------------|----------------|----------------|----------------|----------------|----------------|----------------|-----------------|-----------------|-----------------|-----------------|
-| client-python 1.0  | +              | ✓              | -              | -              |-               |-               |                 |                 |                 |                 |
```
Comment (yliaog) [inline README.md]: should 1.4 be removed, as it is not compatible at all

Comment (roycaihw) [reply README.md]: I agree. Updated

**Pull request merged**